sample/package_sample.py

```diff
@@ -61,6 +62,34 @@ def npmInstallExtensions(appRoot, distPath):
     process.wait()
 
 
+def doVulcanize(appRoot):
+  # Install vulcanize if not available
+  if isProgAvailable(VULCANIZE_CMD) is None:
+    cmd = NPM_CMD + ' install -g ' + VULCANIZE_CMD
+    subprocess.check_call(cmd, shell=True)
+
+  cmd = VULCANIZE_CMD + ' ' \
+       + os.path.join("elements", "elements.html") \
+       + ' --inline-scripts --inline-css --out-html elements.vulcanized.html'
+  print "cmd: " + cmd
```

Comment:
Yeah and I think it's also useful message to users, just like other places in this file calling `print "cmd: " + cmd`